test/e2e/helpers.js

```diff
@@ -33,7 +33,7 @@ async function withFixtures (options, callback) {
     const { driver } = await buildWebDriver(driverOptions)
     webDriver = driver
 
-    // eslint-disable-next-line callback-return
+    // eslint-disable-next-line node/callback-return,node/no-callback-literal
     await callback({
```

Comment:
Hmm. Maybe we should rename this to something that isn't flagged as a callback? :thinking: I don't know if "callback" is the best description anyway.

It's more like a... wrapped function? Like "run this function in a specific environment". Callback implies to me that it's a way to signal when an async task has completed, which this is not.